src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -106,4 +146,49 @@ public void validateCallbackErrorsAndWarnings(final CallbackResponse callbackRes
                 .withWarnings(callbackResponse.getWarnings());
         }
     }
+
+    private List<CallbackRetryContext> buildCallbackRetryContexts(final List<Integer> callbackRetryTimeouts) {
```

Comment:
I think we need to be more uniform with the naming of variables we chose. Otherwise it's complicated to follow the code. In this method parameter we introduce a new name `callbackRetryTimeouts` for the same concept. We have already `callbackRetryIntervalsInSeconds` and `RetryTimeouts`

Reply:
I think there is just a confusion here. It is indeed `callbackRetryTimeouts` that we want here - it is the timeout each consecutive callback should last for. `callbackRetryIntervalsInSeconds` is a different thing - it is the interval each consecutive callback should be be delayed for before execution. The `InSeconds` suffix which could be followed up on similar named variables that do not have it will be addressed as part of 4522 not to delay this any longer.